activesupport/lib/active_support/inflector/transliterate.rb

```diff
@@ -68,26 +68,26 @@ def transliterate(string, replacement = "?".freeze)
     #
     #   parameterize("Donald E. Knuth") # => "donald-e-knuth"
     #   parameterize("^trés|Jolie-- ")  # => "tres-jolie"
-    def parameterize(string, sep = '-')
```

Comment:
I believe this method is also public, so we need to deprecate the positional argument here also